TRANSLATION.md

```diff
@@ -0,0 +1,36 @@
+# Translation Guide
+
+## Add languages
+
+The following tasks are required to add a new language.
+[See available code here](https://github.com/angular/angular/tree/master/packages/common/locales).
+
+- Add  word transration file ​​in `src/assets/i18n/[lang-code].json`.
```

Comment:
typo: `transration`

Reply:
Thanks @maartentibau, I fixed it! (and unknown space)

Reply:
Thanks @maartentibau for keeping an eye on this too! 🙏